src/extensions/behavior/SelectionContext/tooltip.tsx

```diff
@@ -4,17 +4,17 @@ import {Popup, PopupProps} from '@gravity-ui/uikit';
 import {EditorState} from 'prosemirror-state';
 import {EditorView} from 'prosemirror-view';
 
-import {ActionStorage} from '../../../core';
-import {isFunction} from '../../../lodash';
-import {logger} from '../../../logger';
-import {ErrorLoggerBoundary} from '../../../react-utils/ErrorBoundary';
 import {
-    Toolbar,
     ToolbarButtonPopupData,
     ToolbarGroupItemData,
     ToolbarProps,
     ToolbarSingleItemData,
-} from '../../../toolbar';
+} from '../../../bundle/toolbar/types';
```

Comment:
should be imported from `src/toolbar`, not from `src/bundle`
because all extensions is dependencies for `src/bundle`